### ISSUE TITLE
[BUG] kantra newer completes to run / stuck downloading

### ISSUE THREAD

**Issue by maxandersen** [konveyor/kantra#339]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Konveyor version

don't know - using kantra v0.5.3

### Priority

Blocker

### Current Behavior

downloaded kantra for osx and run it and I get stuck with this:

```
kantra analyze --overwrite --output=kantra
LmFaUqLJJGyYaGcG
iJiSrWVpZuEGNJBj
INFO[0000] starting first provider                       provider=java
/opt/podman/bin/podman run -d --rm --name DrdaNnLGYavshAsT --network LmFaUqLJJGyYaGcG -v iJiSrWVpZuEGNJBj:/opt/input/source quay.io/konveyor/java-external-provider:v0.5.3 --port=57434Trying to pull quay.io/konveyor/java-external-provider:v0.5.3...
Getting image source signatures
Copying blob sha256:a22afa8200fd486ebefd72442e04754f56a2d413fead53c6b860742013a675e0
Copying blob sha256:7f4c208fc9d6476429f31af472b905008fc68c5a198c7215fb299d29c916ae28
Copying blob sha256:ee6e912c9b5dd1eccfd8d00e8c068308fe23ad601c13200c01a70d2f7b9c6712
```

### Expected Behavior

that it would complete - and if errors/fail timeout and provide hints to user

### How Reproducible

Always (Default)

### Environment

- OS:OSX

**Comment by dymurray** [comment]
@maxandersen thanks for reporting, looks like this is a bug when `--input` flag isn't set. Can you please rerun with `--input` pointing to the source code of the application you are trying to analyze? I will fix this bug in the meantime

**Comment by dymurray** [comment]
@maxandersen can you confirm that you have followed these instructions? https://github.com/konveyor/kantra/?tab=readme-ov-file#mac-1

This "pause" that you are seeing is podman creating the volume which will be mounted in the container. You should be seeing it in `podman volume ls` after you start kantra. Depending on how big the input dir is and the content it could be taking a few seconds to finish the mounting process and creating the network.

For reference, I'm on linux and I used a 500Gb directory it took ~11 seconds for podman to mount the volume.

I would also recommend rerunning with `--log-level=10` to get more details about what is happening with podman under the hood

**Comment by shawn-hurley** [comment]
Hello @maxandersen I am wondering if you are still hitting this issue, and if we can do anything async or in a call to help unblock you.

**Comment by maxandersen** [comment]
sorry for delay.

I downloaded latest kantra 6 and now hit this:

```
kantra analyze --output kantra --overwrite
INFO[0000]
 --run-local set. running analysis in containerless mode
ERRO[0000] cannot open required path, ensure that container-less dependencies are installed  error="stat /Users/manderse/.kantra: no such file or dirERRO[0000] cannot open required path, ensure that container-less dependencies are installed  error="stat /Users/manderse/.kantra: no such file or dirERRO[0000] failed to validate flags                      error="stat /Users/manderse/.kantra: no such file or directory"
Error: stat /Users/manderse/.kantra: no such file or directory
```

what is container-less dependencies?

**Comment by shawn-hurley** [comment]
@maxandersen There are two things that you can do here:

1. You can use the new containerless workflow for Kantra. To do this, you have to have some things on the filesystem for us to run correctly. You can follow this [guide](https://github.com/konveyor/kantra/blob/main/docs/containerless.md) to get you working in that direction
2. To continue using the container approach, add the option '-run-local false`. 

TLDR: The default switched to containerless because the experience running it that way is better (read faster). This is why we moved it to the default, and I will work with the team to make sure that we add a migration guide to make this more clear on upgrade from 0.5 to 0.6.
